cmd/goal/box.go

```diff
@@ -76,12 +76,12 @@ var appBoxInfoCmd = &cobra.Command{
 		// This reduces confusion of potentially receiving a different box name representation
 		boxNameBytes, err := newAppCallBytes(boxName).Raw()
 		if err != nil {
-			reportErrorf(errorInvalidBoxName, boxName)
+			reportErrorf(errorInvalidBoxName, boxName, err)
 		}
 		if !bytes.Equal(box.Name, boxNameBytes) {
 			reportErrorf(errorBoxNameMismatch, box.Name, boxNameBytes)
 		}
-		reportInfof("Name:  %s", boxName)
+		reportInfof("Name:  %s", encodeBytesAsAppCallBytes(box.Name))
```

Comment:
This is potentially controversial.

I think it's useful for the user to see what their `encoding:value` box name resolves to, but I can also see the argument that we should spit out exactly what they gave us in an attempt to not confuse them.

Reply:
Presumably this function picks an encoding (I'm guessing b64), so I don't know if it's better to show than the encoding the suer chose, which (perhaps) is tied to what they know the key to be, like `int` or `addr`.

Reply:
After more thought, I think I'll revert this for now.

_If_ we think something like this would be useful in the future, we could always add another line called "Name (encoded)" or similar.